x/leveragelp/migrations/v7_migration.go

```diff
@@ -0,0 +1,58 @@
+package migrations
+
+import (
+	"fmt"
+
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/leveragelp/types"
+)
+
+func (m Migrator) V7Migration(ctx sdk.Context) error {
+	// Traverse positions and update lp amount and health
+	// Update data structure
+	positions := m.keeper.GetAllPositions(ctx)
+	pools := m.keeper.GetAllPools(ctx)
+	for _, pool := range pools {
+		m.keeper.DeletePoolPosIdsLiquidationSorted(ctx, pool.AmmPoolId)
+		m.keeper.DeletePoolPosIdsStopLossSorted(ctx, pool.AmmPoolId)
+	}
+	for _, position := range positions {
+		m.keeper.SetSortedLiquidationAndStopLoss(ctx, position)
+	}
+
+	// Liquidate <1.1 positions
+	// Q: What will happen if there won't be enough liquidity to return to users(as health for some positions must be below 1) ? Do we need to fill the pool ?
```

Comment:
@amityadav0 if we dont have enough liquidity in the Testnet pool, we can still close the position as those positions were mostly created for test purposes during the testing phase and as we were actively updating the module functionallities